valhalla/tyr/actor.h

```diff
@@ -21,33 +21,63 @@ class actor_t {
   std::string route(const std::string& request_str,
                     const std::function<void()>* interrupt = nullptr,
                     Api* api = nullptr);
+  std::string route(const std::string& request_str) {
```

Comment:
there has to be a way to do this in the bindings instead of polluting the public c++ interface. i think i might checkout this branch and give it a quick hack to see if i can decypher the correct syntax sugar to make it happen :smile:

Reply:
Yeah, totally agreed, thx!